### ISSUE TITLE
Using Accept header with DOI does not seem to return data consistently

### ISSUE THREAD

**Issue by zmughal** [IQSS/dataverse#2856]
Requesting the [doi:10.7910/DVN/29381](http://dx.doi.org/10.7910/DVN/29381) with
```shell
curl -v -LH "Accept: application/x-research-info-systems" 'http://dx.doi.org/10.7910/DVN/29381'
```
returns a `200 OK` with the response
```
TY  - DATA
T1  - Dataset POVIN_2015_02_v0
AU  - Stanford University Center on Poverty and Inequality; 
PY  - 2015
PB  - Stanford University Center on Poverty and Inequality
UR  - http://dx.doi.org/10.7910/DVN/29381
ER  -
```

while requesting the [doi:10.7910/DVN/MD2UM1](http://dx.doi.org/10.7910/DVN/MD2UM1) with
```shell
curl -v -LH "Accept: application/x-research-info-systems" 'http://dx.doi.org/10.7910/DVN/MD2UM1'
```
returns a `404 Not Found` with the response
```
<html><head><title>Apache Tomcat/7.0.26 - Error report</title><style><!--H1 {font-family:Tahoma,Arial,sans-serif;color:white;background-color:#525D76;font-size:22px;} H2 {font-family:Tahoma,Arial,sans-serif;color:white;background-color:#525D76;font-size:16px;} H3 {font-family:Tahoma,Arial,sans-serif;color:white;background-color:#525D76;font-size:14px;} BODY {font-family:Tahoma,Arial,sans-serif;color:black;background-color:white;} B {font-family:Tahoma,Arial,sans-serif;color:white;background-color:#525D76;} P {font-family:Tahoma,Arial,sans-serif;background:white;color:black;font-size:12px;}A {color : black;}A.name {color : black;}HR {color : #525D76;}--></style> </head><body><h1>HTTP Status 404 - Not Found</h1><HR size="1" noshade="noshade"><p><b>type</b> Status report</p><p><b>message</b> <u>Not Found</u></p><p><b>description</b> <u>The requested resource (Not Found) is not available.</u></p><HR size="1" noshade="noshade"><h3>Apache Tomcat/7.0.26</h3></body></html>
```

**Comment by pdurbin** [comment]
Version 1 of http://dx.doi.org/10.7910/DVN/29381 was published March 6, 2015 which is **before** the migration from DVN 3.x to Dataverse 4 which was in April of 2015.

Version 1 of http://dx.doi.org/10.7910/DVN/MD2UM1 was published on January 11, 2016 which is **after** the migration to Dataverse 4.

I feel like maybe I've heard @mcrosas or @posixeleni or @eaquigley or @scolapasta mention something about this but I'm not sure. If there's an existing issue, let's cross-reference it.

@zmughal thanks for the bug report! It was a pleasure talking to you at http://irclog.iq.harvard.edu/dataverse/2016-01-11

**Comment by sekmiller** [comment]
Not sure what is happening here the identifier is definitely registered with the proper metadata. See 
http://ezid.cdlib.org/id/doi:10.7910/DVN/MD2UM1
Also I can get a proper return via 
curl -v -LH "Accept: application/x-research-info-systems" 'http://dx.doi.org/10.7910/DVN/MTHO5E'
which is a dataset created in Version 4x (11/17/2015)

**Comment by pdurbin** [comment]
We are no longer getting a 404 for [doi:10.7910/DVN/MD2UM1](http://dx.doi.org/10.7910/DVN/MD2UM1)  as originally reported by @zmughal 

```shell
curl -LH "Accept: application/x-research-info-systems" 'http://dx.doi.org/10.7910/DVN/MD2UM1'
```

now returns the following data:

```
TY  - DATA
T1  - Replication Data for: Uneven geographies of organizational practice: explaining the cross-national transfer and adoption of ISO 9000 (with Richard Perkins), Economic Geography 81 (3), 2005, pp. 237-259
AU  - Neumayer, Eric (London School of Economics and Political Science); 
PY  - 2016
PB  - Harvard Dataverse
UR  - http://dx.doi.org/10.7910/DVN/MD2UM1
ER  -
```

@sekmiller you mentioned this might be a time-based thing. That we should give it another day and try again.

**Comment by pdurbin** [comment]
@zmughal is this is indeed an expected delay, how or where would you like or expect this to be documented?

**Comment by zmughal** [comment]
Hmm, I don't know where something like that should be documented. It would make more sense to note it where developers of tools for bibliographic data might see it, but I don't know if that is the role that Dataverse takes.

I only noticed it because I have a small script that I use to quickly retrieve BibTeX data from DOIs.

**Comment by pdurbin** [comment]
@zmughal thanks much. I'm not actually sure if there's anything actionable for this issue so I'm passing it to QA for comment or to close.

**Comment by pdurbin** [comment]
I reached out to EZID folks via private email (and cc'd @sekmiller ) and there are more hops in this pipeline than I realized:

Dataverse -> EZID -> DataCite -> other hops -> doi.org

The suggestion I was given was "unless the delay is more than 24hrs I would hesitate to bring the issue up with [DataCite]".

Makes sense. @kcondon I'm moving this to 4.2.3 to get your feedback. I still think we can probably close this but I'd like to know what you think.
